CMakeLists.txt

```
@@ -83,3 +85,8 @@ if(CMAKE_RUN_CLANG_TIDY)
     -header-filter='.*'
     -checks=${CLANG_TIDY_CHECKS})
 endif()
+
+if((${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME} OR ${PROJECT_NAME}_BUILD_TESTING) AND BUILD_TESTING)
+    include(CTest)
```

```suggestion
    include(CTest)
    enable_testing()
```

is necessary if builded as dependency